Subject: [PATCH] ExtService: Fix no icon shown for data limit notification

When USB connects and data warning and limit are enabled, the icon
for data limit is not shown in status bar after data limit reaches.

The reason is that when data limit notification reaches, the total
number of notifications for package android becomes 4, then will
auto-group, the group summary will re-use the firstly enqueued
notification's channel and importance, USB Charging notification is
the firstly enqueued one whose channel has IMPORTANT_MIN importance,
SystemUI will not show the icon for this unimportant notification,
it's considered as ambient notification in SystemUI.

For notifications in package android, they have different icons and
importance, so will not auto-group them.

diff --git a/packages/ExtServices/src/android/ext/services/notification/Ranker.java b/packages/ExtServices/src/android/ext/services/notification/Ranker.java
index 2ce667c..9e3ac58 100644
--- a/packages/ExtServices/src/android/ext/services/notification/Ranker.java
+++ b/packages/ExtServices/src/android/ext/services/notification/Ranker.java
@@ -64,6 +64,7 @@
                 // Not grouped by the app, add to the list of notifications for the app;
                 // send bundling update if app exceeds the autobundling limit.
                 synchronized (mUnbundledNotifications) {
+                    if ("android".equals(sbn.getPackageName())) return;
                     Map<String, LinkedHashSet<String>> unbundledNotificationsByUser
                             = mUnbundledNotifications.get(sbn.getUserId());
                     if (unbundledNotificationsByUser == null) {
@@ -211,4 +212,4 @@
                 null, user);
     }
 
-}
\ No newline at end of file
+}
